en-US/firefox/products/developer-quantum.lang

```diff
@@ -0,0 +1,191 @@
+## NOTE: demo page available at https://bedrock-demo-craigcook.us-west.moz.works/firefox/developer/
+## URL: https://bedrock-demo-craigcook.us-west.moz.works/%LOCALE%/firefox/developer/
+## NOTE: Product and tool names remain in English
```

Comment:
Why would tool names remain in English?

Reply:
Wrong format (note, url, note), and definitely wrong comment